src/azul/deployment.py

```diff
@@ -88,10 +98,13 @@ def es_endpoint(self) -> Netloc:
 
     @property
     def lambda_env(self) -> Mapping[str, str]:
-        return config.lambda_env(self.es_endpoint)
+        return {
+            **config.lambda_env(self.es_endpoint),
+            'AZUL_SERVICE_API_GATEWAY_ID': aws.api_gateway_id('service', validate=True)
+        }
 
     def get_lambda_arn(self, function_name, suffix):
-        return f"arn:aws:lambda:{aws.region_name}:{aws.account}:function:{function_name}-{suffix}"
```

Comment:
There may be more. Fix them all and move to separate commit if you haven't done so already.

Reply:
That was the only one I found. I will more to a separate commit.